pybossa/view/bloomberg.py

```diff
@@ -104,3 +106,10 @@ def handle_bloomberg_response():
         current_app.logger.exception('BSSO login error')
         flash(gettext('We were unable authenticate and log you into an account. Please contact a Gigwork administrator.'), 'error')
         return redirect(url_for('home.home'))
+        
+
+def get_user_data_access_level(firm_num):
+    """Reads firm id to user type mappings from settings_upref_mdata and
+    returns the access type"""
+    return ['L2'] if firm_num and int(firm_num) in app_settings.upref_mdata.firm_id_to_type_mapping.keys() else ['L4']
```

Comment:
L2 access to be only set under private instance, whereas current logic could set it under public instance in case of firm id not configured.

Reply:
I am changing this so that L2 can only be returned for the firms in the list in Private. Public will default everyone to L4.